### PR TITLE
Enable mustache ci and comment mustache files

### PR DESCRIPTION
**Pull request by smbader**
Turn on mustache ci actions and add missing comments from mustache files.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -75,8 +75,6 @@ jobs:
         env:
           DB: ${{ matrix.database }}
           MOODLE_BRANCH: ${{ matrix.moodle-branch }}
-          IGNORE_NAMES: 'templates/mobile_view_page.mustache'
-          MUSTACHE_IGNORE_NAMES: 'templates/mobile_view_page.mustache'
```
Comment (jrchamp) [inline .github/workflows/ci.yml:79]: It seems like the format is fine, so I'm not sure how other people are convincing the HTML validator of that. Maybe @dpalou knows?

For now, let's try ignoring the files and we can revisit.
```yml
          MUSTACHE_IGNORE_NAMES: 'mobile_view_page_latest.mustache,mobile_view_page_ionic3.mustache'
```

File: templates/mobile_view_page_ionic3.mustache [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+        "available": true,
+        "status": "Finished",
+        "start_time": "Tuesday, June 29, 2021, 1:30 PM",
+        "duration": "1 hour",
+        "option_audio": "Computer audio and Telephone",
+        "cmid": 3,
+        "courseid": 3
+    }
+}}
+
```
Comment (jrchamp) [inline templates/mobile_view_page_ionic3.mustache:48]: The other examples that I've seen do not have an empty line here, so I'll suggest removal.

```suggestion
```

File: templates/mobile_view_page_latest.mustache [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+        "available": true,
+        "status": "Finished",
+        "start_time": "Tuesday, June 29, 2021, 1:30 PM",
+        "duration": "1 hour",
+        "option_audio": "Computer audio and Telephone",
+        "cmid": 3,
+        "courseid": 3
+    }
+}}
+
```
Comment (jrchamp) [inline templates/mobile_view_page_latest.mustache:48]:
```suggestion
```

**Review by jrchamp — COMMENTED** [review]
This seems like an improvement overall. Maybe HQ has some ideas?

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -75,8 +75,6 @@ jobs:
         env:
           DB: ${{ matrix.database }}
           MOODLE_BRANCH: ${{ matrix.moodle-branch }}
```
Comment (jrchamp) [inline .github/workflows/ci.yml:77]:
```suggestion
          MOODLE_BRANCH: ${{ matrix.moodle-branch }}
          MUSTACHE_IGNORE_NAMES: 'mobile_view_page_latest.mustache,mobile_view_page_ionic3.mustache'
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dpalou) [reply .github/workflows/ci.yml:79]: Sorry, I don't know much about the HTML validator. You might want to ask in the [Telegram Moodle Dev Chat](https://docs.moodle.org/dev/Chat).

**Review by jrchamp — APPROVED** [review]
Other than some extraneous warnings from the HTML validator, this passed the tests and improved the template documentation. Thanks!

**Pull request merged**